src/text.tsx

```diff
@@ -126,7 +126,7 @@ export const Text: React.FC<TextProps> = ({
             id,
             role,
             'aria-level': ariaLevel,
-            ...getPrefixedDataAttributes(dataAttributes),
+            ...getPrefixedDataAttributes(dataAttributes, 'Text'),
```

Comment:
this component is too low level, I think the data-component-name won't give any value in TextX components